src/main/webapp/index-lti.vm

```diff
@@ -103,7 +103,17 @@
         <div ng-include src="'assets-lti/views/friends.html'"></div>
       </div>
     </div>
-
+    <a href="" style="position:fixed;right:0;top:60px;width:30px" ng-click="showDebug = !showDebug">&nbsp;</a>
```

Comment:
add aria-hidden="true" attribute to link, might as well add it to the div too.

Reply:
Done